lib/eChronicler_web/router.ex

```diff
@@ -18,6 +18,9 @@ defmodule EChroniclerWeb.Router do
     pipe_through :browser
 
     get "/", PageController, :index
+    get "/hello", HelloController, :index
+    get "/hello/:messenger", HelloController, :show
+    get "/journal_entries", JournalEntryController, :index
```

Comment:
I would delete the two `HelloController` routes. I also would like to see this list of journal entries at the root route (i.e. `https://echronicler.herokuapp.com/` not `https://echronicler.herokuapp.com/journal_entries/`).